content/useStyles: a hook to manage your styles - Agustín Prieto - EN.md

```diff
@@ -0,0 +1,65 @@
+![Banner React Native](images/useStylesBanner.jpg)
+
+# useStyles: a hook to manage styles in React Native
+
+¿Does anyone know how to properly manage styles in React Native? I don't. 
+
+I Have been working with this platform for a couple of years now and couldn't get a concise way of using styles. This last year I've tried some libraries with novel concepts as Styled Components, but I couldn't get used to them. As a result of this dissatisfaction with the current alternatives, useStyles was born.
```

Comment:
I don't know if `platform` is the correct term to referring a framework like react native.
![Screen Shot 2020-10-14 at 11 39 04](https://user-images.githubusercontent.com/9642510/96004532-e7231100-0e11-11eb-8e75-66e8960086ca.png)
Also, I think is better for redaction and for "telling the story" the `platform` term, but as always you have to choose between have an article super-technical vs a nice and super agile article for everyone.